libsession/src/main/java/org/session/libsession/messaging/sending_receiving/MessageReceiverHandler.kt

```diff
@@ -549,11 +577,14 @@ private fun MessageReceiver.handleClosedGroupMemberLeft(message: ClosedGroupCont
         // admin left the group of linked device left the group
         disableLocalGroupAndUnsubscribe(groupPublicKey, groupID, userPublicKey)
     } else {
-        val isCurrentUserAdmin = admins.contains(userPublicKey)
+        //val isCurrentUserAdmin = admins.contains(userPublicKey)
```

Comment:
Let's remove this rather than just commenting it out. Just to keep things clean and organized!

Reply:
Oh yeah, missed that one ^^